backend/alembic/versions/0002_create_skills.py

```diff
@@ -0,0 +1,84 @@
+"""Init database
+
+Revision ID: 0002
+Revises: 0001
+Create Date: 2022-09-18 00:00:00
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "0002"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+UPGRADE_SQL = """
+CREATE TABLE IF NOT EXISTS `role` (
+    `Role_ID` int NOT NULL AUTO_INCREMENT,
+    `Role_Name` varchar(20) NOT NULL,
+    PRIMARY KEY (`Role_ID`)
+) DEFAULT CHARSET=utf8;
```

Comment:
These tables seem to conflict with 0001 (i.e. creating tables of the same name). Would be better to either alter the existing tables or drop them then recreate